storage/innobase/trx/trx0sys.cc

```diff
@@ -361,7 +361,17 @@ trx_sys_update_wsrep_checkpoint(
             read_wsrep_xid_uuid(xid, xid_uuid);
             if (!memcmp(xid_uuid, trx_sys_cur_xid_uuid, 8))
             {
-                ut_ad(xid_seqno > trx_sys_cur_xid_seqno);
+				/* We have the case when REPLACE INTO ... SELECT or
```

Comment:
Just to make it look uniform with the rest of the function, can we use whitespaces here?